diff --git a/target/linux/ipq40xx/Makefile b/target/linux/ipq40xx/Makefile
index c75f3d51..ab303073 100644
--- a/target/linux/ipq40xx/Makefile
+++ b/target/linux/ipq40xx/Makefile
@@ -19,8 +19,6 @@ DEFAULT_PACKAGES += \
 	kmod-leds-gpio kmod-gpio-button-hotplug swconfig \
 	kmod-ath10k-ct wpad-openssl \
 	kmod-usb3 kmod-usb-dwc3 ath10k-firmware-qca4019-ct \
-	automount autosamba luci-app-ipsec-vpnd v2ray shadowsocks-libev-ss-redir shadowsocksr-libev-server luci-app-music-remote-center \
-	luci-app-unblockmusic luci-app-cpufreq luci-app-zerotier luci-app-xlnetacc ca-certificates kmod-fs-cifs kmod-nls-utf8 \
-	htop fdisk e2fsprogs
+	automount
 
 $(eval $(call BuildTarget))
